src/components/metadataEditor/metadataEditor.js

```diff
@@ -244,14 +244,14 @@ import 'flexStyles';
         if (parentId) {
             reload(context, parentId, item.ServerId);
         } else {
-            import('appRouter').then(({default: appRouter}) => {
+            import('../appRouter').then((appRouter) => {
```

Comment:
```suggestion
            import('../appRouter').then(({ appRouter }) => {
```

Reply:
why is this one `{ foo }` and the others `{default: foo}` ?

Reply:
`appRouter` module has no `default` export - it exports `const appRouter`
https://github.com/jellyfin/jellyfin-web/blob/2c899c93dff42d7cefc1c2cbeb0e63eaf5d02b8b/src/components/appRouter.js#L838

Some modules, on the other hand, have `default` export and nothing more. In this case, we can only import `default`.

Reply:
we could always modify which exports are done per module, but that can be done in a future iteration.

imported in the beginning of the module.